docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: "3.9"
+services:
+  hugo:
+    # The version here must be kept in sync with the one used at .github/workflows/main.yml
+    image: klakegg/hugo:0.80.0-ext-ubuntu
+    command: server
```

Comment:
```suggestion
    command: server --appendPort=false --baseURL=/
```
How about this? When using GitHub Codespaces in a docker in docker environment, ``hugo server`` alone will not load css and js when port forwarded, causing design corruption. 

The GitHub Codespaces default docker image includes hugo, but I think it's worth using docker-compose to intentionally match the version.
<img width="1047" alt="image" src="https://user-images.githubusercontent.com/15963767/219660293-156531f1-0023-42c5-af9d-32579f1e63e0.png">

Reply:
I wrote about this in the i18n document as well. If you have a better way, let me know!
https://github.com/InnerSourceCommons/innersourcecommons.org/blob/master/i18n.md#debug-with-github-codespaces

Reply:
Hello @yuhattor , I was not aware of this limitation, thank you for the enhancement!
Supporting codespaces is very useful.